src/mon/MgrStatMonitor.cc

```diff
@@ -79,7 +79,9 @@ void MgrStatMonitor::update_from_paxos(bool *need_bootstrap)
   version = get_last_committed();
   dout(10) << " " << version << dendl;
   bufferlist bl;
-  get_version(version, bl);
+  int err = get_version(version, bl);
```

Comment:
`MgrStatMonitor` does not necessary have its epoch if `version = 0`, because it does not create a pending epoch in `create_initial()`.